src/blocks/Image/ZestyImage.js

```diff
@@ -0,0 +1,38 @@
+import FillerContent from 'components/globals/FillerContent';
+
+/**
+ *
+ * @param {string} src - (required!) remote url of the image
+ * @param {string} alt - (required!) alternative text of the image
+ * @param {object} options - (required!) pass optimization options as key values pairs, see parameters options here -> https://zesty.org/services/media-storage-micro-dam/on-the-fly-media-optimization-and-dynamic-image-manipulation#auto-optimize-image-jpg-auto
+ * @param {object} style - (optional) custom inline css style
+ * @param {string} loading - {optional} options - lazy | eager
+ * @param {object} attributes - (optional) any custom attributes you want to pass must be in key value pairs
+
+ * @returns react <img/> component
+ */
+
+const ZestyImage = ({ options, src, alt, loading, style, attributes }) => {
+  /* Taking the options object and converting it into a query string. */
+  const imageUrl = Object.entries(options).reduce(
+    (acc, item, idx) => {
+      const newUrl = `${acc}${idx === 0 ? '?' : '&'}${item[0]}=${item[1]}`;
+      return newUrl;
+    },
+    src ? src : FillerContent.photos[0].src,
+  );
+
+  return (
+    <img
+      width={options.width}
```

Comment:
options.width, options.height... these options need a default or need to be required by the component

Reply:
a default could be "auto"

Reply:
@ardeay  for this I made it required cause it solves one of the common issues in our website based on lighthouse report. For example this one 

![image](https://user-images.githubusercontent.com/61284357/188630442-2343948c-a519-42ea-9d65-7d3c8ee9f85b.png)

* Aspect ratio actual is the image returned from our CDN
* Aspect ratio displayed is the one displayed on the website

if these two don't match we will get this error on lighthouse or we can possibly penalize in terms of SEO as well, so keeping it required, we can ensure that we have requested the same width and height from CDN and display the same on the front end.

Reply:
Its not required because you pass the params in an object. The component should check those params exist on the options object, or width and height should be deconstrcuted props